libnmstate/nm/device.py

```diff
@@ -187,45 +179,6 @@ def _modify_callback(src_object, result, user_data):
         _activate_async(src_object)
 
 
-def _requires_activation(dev, connection_profile):
-    if StrictVersion(nmclient.nm_version()) < StrictVersion(
-        '1.18'
-    ) and _mtu_changed(dev, connection_profile):
-        logging.debug(
-            'Device reapply does not support mtu changes, '
-            'fallback to device activation: dev=%s',
-            dev.get_iface(),
-        )
-        return True
-    return False
-
-
-def _mtu_changed(dev, connection_profile):
-    wired_setting = connection_profile.get_setting_wired()
-    configured_mtu = wired_setting.props.mtu if wired_setting else None
-    if configured_mtu:
-        current_mtu = int(dev.get_mtu())
-        return configured_mtu != current_mtu
-    return False
-
-
-def _ipv6_changed(dev, connection_profile):
```

Comment:
While it is correct to remove this, it does not belong into this commit, since it is a follow-up on #641

Reply:
Oops, yes you are right. Thanks!

Reply:
Fixed!